__tests__/app.test.js

```diff
@@ -393,3 +394,62 @@ describe("GET /api/users", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: should return an array of articles with order by asc", () => {
+    return request(app)
+      .get("/api/articles?order=asc")
+      .expect(200)
+      .then(({ body: { articles } }) => {
```

Comment:
It might be nice to have an assertion for the sort order here, it would also be great to check the length before you start looping to prevent any false positives.